pubsub/google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -66,6 +91,18 @@ def __init__(self, batch_settings=(), batch_class=thread.Batch, **kwargs):
         self._batch_lock = threading.Lock()
         self._batches = {}
 
+    @property
+    def target(self):
```

Comment:
@lukesneeringer This doesn't address the issue I had with it. It will still get re-computed every time this property is accessed (i.e. at run time, rather than import time).